benchmark/benchmark.py

```diff
@@ -79,14 +97,19 @@ def profile(func, identifier):
 
 
 def cli():
-    "Read the commandline parameters, perform the requested action."
-    # import pdb; pdb.set_trace()
+    """Parse the command line arguments, perform the requested action.
+    """
+    parser = argparse.ArgumentParser(description='PennyLane benchmarking tool')
+    parser.add_argument('--version', action='version', version=__version__)
+    #parser.add_argument('-v', '--verbose', action='store_true', help='verbose mode')
+    parser.add_argument('-d', '--device', type=lambda x: x.split(','), default=['default.qubit'], help='comma-separated list of devices to run the benchmark on (default: %(default)s)')
```

Comment:
Hmm, didn't know about this. I guess `argparse` looks for specific substrings and replaces them?

Reply:
The `type` argument is essentially a conversion function that's applied on the string received from commandline. Here it's `device1,device2,device3`, commas with no spaces separating the device names.